zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/annotation/PostMappingStringKeys.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.annotation;
+
+import io.camunda.zeebe.gateway.rest.RequestMapper;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@RequestMapping(method = RequestMethod.POST)
+public @interface PostMappingStringKeys {
```

Comment:
❓ How about we use a generic name here like `CamundaRestPostMapping` or something similar? This would not make this so much about the string keys but simply create a default annotation that we can keep in the future. It simply provides all the defaults you usually need for an endpoint.

Reply:
Sounds good to me, I'll update it.